src/sentry/static/sentry/app/api.jsx

```diff
@@ -174,6 +181,7 @@ export class Client {
       });
       throw err;
     }
+
     const method = options.method || (options.data ? 'POST' : 'GET');
     let data = options.data;
```

Comment:
This LOC and `data = JSON.stringify(data);` (at the next few lines of code later)  may need to be moved to the top since the span consumes this.